src/test_utils.ts

```diff
@@ -36,7 +44,115 @@ export async function renderAndWait(
   await act(async () => {
     renderResult = render(ui, options);
     // Silence warnings caused by async DOM updates.
-    await delay(100);
+    await delay(200);
```

Comment:
might be able to use `jest.runAllTimers()` here (for another PR), I'm concerned that we'll need to keep pushing this up to fit the slowest render, slowing down all other tests.

https://jestjs.io/docs/en/timer-mocks

Reply:
👍 love your work